subprojects/plugins/src/main/resources/org/gradle/api/internal/plugins/windowsStartScript.txt

```diff
@@ -44,11 +44,11 @@ set JAVA_EXE=java.exe
 %JAVA_EXE% -version >NUL 2>&1
 if %ERRORLEVEL% equ 0 goto execute
 
-echo.
-echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
+echo. 1>&2
+echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH. 1>&2
+echo. 1>&2
```

Comment:
Seems like an old issue.  I just want to point out `echo.` is not the best option to output a blank line – [_Echo a blank line_](https://ss64.com/nt/echo.html#blank) (SS64.com):

>  To produce an empty line, without displaying the echo status, use the Echo command immediately followed with a colon (delimiter):
>
>   ```
>   Echo First Line
>   Echo:
>   ```
>   ...
>
>   Other command delimiters are available but have buggy [side effects](https://www.dostips.com/forum/viewtopic.php?f=3&t=774&start=0), for example `Echo.` will search for a file named "echo" (or echo.exe) in the current directory, if such a file is found that will raise an error. If the 'echo' file does not exist then the command does work, but this still makes `Echo.` slightly slower than `echo:` (In recent builds of WIndows 10 `Echo.` will always throw an error.)